controllers/apply/form-router/index.js

```diff
@@ -64,10 +70,22 @@ function initFormRouter({
         next();
     }
 
+    function setFormBrand(req, res, next) {
+        const validBrand = allowedBrands.find(
```

Comment:
If using an object for these this could become `const validBrand = allowedBrands[req.query.brand]` or `const validBrand = get(allowedBrands, req.query.brand)`